docs/proposals/dedicated-shipwright-operator.md

```diff
@@ -0,0 +1,212 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+---
+title: dedicated-shipwright-operator
+authors:
+  - "@adambkaplan"
+reviewers:
+  - "@SaschaSchwarze0"
+  - "@zhangtbj"
+  - "@gabemontero"
+approvers:
+  - "@qu1queee"
+  - "@sbose78"
+creation-date: 2021-02-03
+last-updated: 2021-02-03
+status: provisional
+---
+
+# Dedicated Shipwright Operator
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [docs](/docs/)
+
+## Open Questions [optional]
+
+TBD
+
+## Summary
+
+Create a separate operator container image that configures and deploys Shipwright Builds.
+The operator will consume a Custom Resource Definition (CRD) which defines how Shipwright Builds should be configured.
+The operator then reconciles the CRD spec with Shipwright Build's controller deployment.
+
+The design has been inspired by the upstream [Tekton operator](https://github.com/tektoncd/operator).
+
+## Motivation
+
+This proposal resolves issue [#310](https://github.com/shipwright-io/build/issues/310) by creating a separate operator container image, which is responsible for installing Shipwright Builds on the cluster.
+This will be a true operator, initialized by operator-sdk to create a bundle image.
+The bundle image will include CRDs that control how Shipwright Builds are configured.
```

Comment:
This part needs a little more clarity:

Would the OLM manifest/CSV reference the `Build`, `BuildRun` and `BuildStrategy` CRDs? 
Or would the controller create/update these CRDs the way Tekton operator does?

Reply:
Per discussion this AM, I updated the proposal to have OLM install the Build API CRDs.

I did call out that this is not a "permanent" decision - in the future we can have the operator manage the Build API CRDs. And furthermore we can add RBAC to ensure that only shiwpright.io CRDs are managed (not a global "install any CRD").

Reply:
Sounds great, thank you!